Gemfile.lock

```diff
@@ -0,0 +1,169 @@
+GEM
```

Comment:
Many gems in Gemfile are not locked. Please lock them up with appropriate versions.